chart/values.yaml

```diff
@@ -627,6 +627,8 @@ obs:
     enabled: true
     # -- Log level for callhome
     logLevel: "info"
+    # -- Save the report
```

Comment:
```suggestion
```

Don't think we should document this, makes no sense for an end-user?

Reply:
removed